frontend/packages/pipelines-plugin/package.json

```
@@ -24,8 +24,16 @@
       "yamlTemplates": "src/templates/pipelines.ts",
       "catalog": "src/components/catalog",
       "topology": "src/topology",
-      "pipelineTabbedPage": "src/components/pipelines/PipelineTabbedPage.tsx",
-      "repositoryListComponent": "src/components/repository/list-page"
+      "pipelinesComponent": "src/components/pipelines",
+      "pipelineRunsComponent": "src/components/pipelineruns",
+      "taskRunsComponent": "src/components/taskruns",
+      "repositoryComponent": "src/components/repository",
+      "clusterTasksComponent": "src/components/cluster-tasks",
+      "tasksComponent": "src/components/tasks",
+      "conditionsComponent": "src/components/conditions",
+      "pipelineComponent": "src/components",
+      "pipelinesListsComponent": "src/components/pipelines-lists",
+      "triggersListsComponent": "src/components/triggers-lists"
```

Instead have a index.ts in `src/components`  that exports all these components and expose it as  `pipelineComponents` like below.
```suggestion
      "pipelineComponents": "src/components",
```